### PR TITLE
NMS-13883: Add graphics to flows documentation

### PR DESCRIPTION
**Pull request by Bonrob2**
NMS-13883: Add graphics to flows documentation: added updated flows diagrams for flows with Minion and Sentinel sections.

### All Contributors

* [ ] Have you read our [Contribution Guidelines](https://github.com/OpenNMS/opennms/blob/develop/CONTRIBUTING.md)?
* [ ] Have you (electronically) signed the [OpenNMS Contributor Agreement](https://cla-assistant.io/OpenNMS/opennms)?

### Contribution Checklist

* Please [make an issue in the OpenNMS issue tracker](https://issues.opennms.org) if there isn't one already.<br />Once there is an issue, please:
  1. update the title of this PR to be in the format: `${JIRA-ISSUE-NUMBER}: subject of pull request`
  2. update the JIRA link at the bottom of this comment to refer to the real issue number
  3. prefix your commit messages with the issue number, if possible
  4. once you've created this PR, please link to it in a comment in the JIRA issue
  Don't worry if this sounds like a lot, we can help you get things set up properly.
* If this is a new or updated feature, is there documentation for the new behavior?
* If this is new code, are there unit and/or integration tests?
* If this PR targets a `foundation-*` branch, does it try to avoid changing files in `$OPENNMS_HOME/etc/`?

### What's Next?

A PR should be assigned at least 2 reviewers.  If you know that someone would be a good person to review your code, feel free to add them.

If you need help making additions or changes to the documentation related to your changes, please let us know.

In any case, if anything is unclear or you want help getting your PR ready for merge, please don't hesitate to say something in the comments here,
or in [the #opennms-development chat channel](https://chat.opennms.com/opennms/channels/opennms-development).

Once reviewer(s) accept the PR and the branch passes continuous integration, the PR is eligible for merge.

At that time, if you have commit access (are an OpenNMS Group employee or a member of the OGP) you are welcome to merge the PR when you're ready.
Otherwise, a reviewer can merge it for you.

Thanks for taking time to contribute!

### External References

* JIRA (Issue Tracker): http://issues.opennms.org/browse/NMS-13883

### CODE REVIEW

**Review by mmahacek — DISMISSED** [review]
LGTM

**Comment by Bonrob2** [comment]
@cgorantla , would you be able to verify that these graphics are correct?

**Comment by cgorantla** [comment]
There is a minor deviation: SFlow still uses Bson from parser to Adapter.  Otherwise graph looks good to me.

**Comment by Bonrob2** [comment]
> There is a minor deviation: SFlow still uses Bson from parser to Adapter. Otherwise graph looks good to me.

Thanks. Will update.

**Comment by Bonrob2** [comment]
@cgorantla updated and ready for your review.

**Comment by cgorantla** [comment]
> @cgorantla updated and ready for your review.

`SFlow` still seems to be on the left side with `Netflow5` which still seems to be wrong. I'm okay with omitting that minor detail of `BSON` if it gets confusing on the graph. 


Also on the sentinel graph,  `adapters` also run on Sentinel not just the enrichment.

**Comment by Bonrob2** [comment]
> > @cgorantla updated and ready for your review.
> 
> `SFlow` still seems to be on the left side with `Netflow5` which still seems to be wrong. I'm okay with omitting that minor detail of `BSON` if it gets confusing on the graph.
> 
> Also on the sentinel graph, `adapters` also run on Sentinel not just the enrichment.

My bad. I missed the sFlow in that part. Will update again.

**Comment by Bonrob2** [comment]
@cgorantla Third time lucky?

**Comment by cgorantla** [comment]
One more thing that I missed earlier, need to update `flow_integration_sentinel.png`.  Netflow adapters also run on Sentinel not just the enrichment.

File: docs/modules/operation/pages/flows/distributed.adoc [hunk 1]
```diff
@@ -1,10 +1,14 @@
 
 [[flows-remote]]
-= Distributed/Remote Flows
+= Distributed/Remote Flows with Minion
```
Comment (indigo423) [inline docs/modules/operation/pages/flows/distributed.adoc:3]: Just as an idea for the title: "Using Minions as a flow collector", the term "flow collector" is well known in this domain.

File: docs/modules/operation/pages/flows/sentinel/sentinel.adoc [hunk 2]
```diff
@@ -1,6 +1,6 @@
 
 [[flows-scaling]]
-= Scale Flows Data
+= Scale Flows Data with Sentinel
```
Comment (indigo423) [inline docs/modules/operation/pages/flows/sentinel/sentinel.adoc:3]: Suggestion for the title: "Scale flow processing with Sentinel"

**Review by indigo423 — APPROVED** [review]
Just minor ideas and nothing which blocks it for me to merge it :) Thanks a lot for digging through this topic!

**Comment by Bonrob2** [comment]
> One more thing that I missed earlier, need to update `flow_integration_sentinel.png`. Netflow adapters also run on Sentinel not just the enrichment.

Updated, thanks for the help @cgorantla

**Review by cgorantla — APPROVED** [review]

**Review by mmahacek — APPROVED** [review]
I've got no objections. 👍

**Pull request merged**